src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -789,7 +789,7 @@ public class TraceEventListener : EventListener
 
             protected override void OnEventSourceCreated(EventSource eventSource)
             {
-                if (eventSource.Name.Equals("Microsoft.Data.SqlClient.EventSource"))
+                if (eventSource.Name.StartsWith("Microsoft.Data.SqlClient"))
```

Comment:
By adding support for more than one event source, I suggest including the event source name in the list to verify if all events run per event source.

Also, could you add a test for `AKVEventSource`, please?